samples/R7/test-production.sql

```diff
@@ -0,0 +1,40 @@
+---------- R7: Sahl ----------
+/* GET PLAYER STATS */
+SELECT * FROM Player WHERE pid = 120;
+
+SELECT assists, points, games, season, abbrev, tname 
+FROM PlayerStats NATURAL JOIN Team
+WHERE pid = 120
+ORDER BY season DESC;
```

Comment:
This might also give duplicate sorting, maybe sort by pid as secondary? (can also be an "optimization for prod)

Reply:
Don't forget to update the feature too!

Reply:
I don't see how duplicate sorting would occur
Maybe I am misunderstanding what u are saying

Reply:
Some players are in multiple teams in the same season. For example, if you do a query on pid 5026, they were in MIN in 2023 and also in DAL in 2023. This causes undefined order for MIN and DAL